src/main/java/uk/gov/justice/probation/courtcaseservice/jpa/entity/CourtEntity.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.justice.probation.courtcaseservice.jpa.entity;
```

Comment:
Probably not important at this point but you will probably want to store an external Court ID here from Delius or the Court service?

Reply:
It's just a starter for 10, we'll add that in when we know more about the api.